tutorials/tour/_posts/2017-02-13-unified-types.md

```diff
@@ -11,15 +11,15 @@ next-page: classes
 previous-page: basics
 ---
 
-In Scala, all values are instances of a class, including numerical values and functions. The diagram below illustrates the class hierarchy.
+In Scala, all values are instances of a class, including numerical values and functions. The diagram below illustrates a subset the class hierarchy.
```

Comment:
already settled I guess, but I agree that it's better to say "type" instead of "class" in this (and many other) contexts